### PR TITLE
Add ScalapackFx package

### PR DESCRIPTION
**Pull request by awvwgk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by awvwgk** [comment]
@aradi @bhourahine Let me know if you want to join as maintainers on the ScalapackFx feedstock.

**Comment by bhourahine** [comment]
@awvwgk, sure, can do.

**Comment by aradi** [comment]
@awvwgk I am definitely in, thanks a lot for setting it up!

**Comment by awvwgk** [comment]
@conda-forge/staged-recipes This Fortran MPI-enabled package for Linux and OSX is ready for review.

File: recipes/scalapackfx/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+   "-DSCALAPACK_LIBRARY=scalapack"
+   "-GNinja"
+   ".."
+)
+
+mkdir -p _build
+pushd _build
+cmake "${cmake_options[@]}"
+
+ninja all install
```
Comment (ocefpaf) [inline recipes/scalapackfx/build.sh:18]: Is there a make check, make test, or ctest that we can run here?

**Review by ocefpaf — APPROVED** [review]
Minor question about tests that can be addressed in the feedstock. Thanks!

**Pull request merged**